tap_github/tap.py

```diff
@@ -49,7 +46,7 @@ class TapGitHub(Tap):
         ),
         th.Property(
             "rate_limit_buffer",
-            th.ArrayType(th.IntegerType),
+            th.IntegerType,
```

Comment:
@ericboucher seeing how this is used I think it's safe to assume it's an integer and not an array